tests/integration/servicemesh/smmr_test.go

```diff
@@ -41,20 +48,29 @@ func TestSMMR(t *testing.T) {
 			applyVirtualServiceOrFail(ctx, namespaceA, namespaceGateway, "a")
 			applyVirtualServiceOrFail(ctx, namespaceB, namespaceGateway, "b")
 
-			if err := maistra.ApplyServiceMeshMemberRoll(ctx, namespaceGateway, namespaceA); err != nil {
-				ctx.Fatalf("failed to create ServiceMeshMemberRoll: %s", err)
-			}
-			verifyThatIngressHasVirtualHostForMember(ctx, "a")
+			ctx.NewSubTest("JoiningMesh").Run(func(t framework.TestContext) {
+				if err := maistra.ApplyServiceMeshMemberRoll(ctx, namespaceGateway, namespaceA); err != nil {
+					t.Fatalf("failed to create ServiceMeshMemberRoll: %s", err)
+				}
+				verifyThatIngressHasVirtualHostForMember(t, "a")
 
-			if err := maistra.ApplyServiceMeshMemberRoll(ctx, namespaceGateway, namespaceA, namespaceB); err != nil {
-				ctx.Fatalf("failed to add member to ServiceMeshMemberRoll: %s", err)
-			}
-			verifyThatIngressHasVirtualHostForMember(ctx, "a", "b")
+				if err := maistra.ApplyServiceMeshMemberRoll(ctx, namespaceGateway, namespaceA, namespaceB); err != nil {
+					t.Fatalf("failed to add member to ServiceMeshMemberRoll: %s", err)
+				}
+				verifyThatIngressHasVirtualHostForMember(t, "a", "b")
 
-			if err := maistra.ApplyServiceMeshMemberRoll(ctx, namespaceGateway, namespaceB); err != nil {
-				ctx.Fatalf("failed to create ServiceMeshMemberRoll: %s", err)
-			}
-			verifyThatIngressHasVirtualHostForMember(ctx, "b")
+				if err := maistra.ApplyServiceMeshMemberRoll(ctx, namespaceGateway, namespaceB); err != nil {
+					t.Fatalf("failed to create ServiceMeshMemberRoll: %s", err)
+				}
+				verifyThatIngressHasVirtualHostForMember(t, "b")
+			})
+
+			ctx.NewSubTest("RouteCreation").Run(func(t framework.TestContext) {
+				if err := maistra.EnableIOR(t); err != nil {
```

Comment:
I don't enable IOR for all tests in the suite to increase coverage and be sure that Maistra works fine when IOR is enabled as well as disabled.